src/allmydata/web/status.py

```diff
@@ -673,90 +673,129 @@ def render_progress(self, ctx, data):
     def render_status(self, ctx, data):
         return data.get_status()
 
-class RetrieveStatusPage(rend.Page, RateAndTimeMixin):
-    docFactory = getxmlfile("retrieve-status.xhtml")
 
-    def __init__(self, data):
-        rend.Page.__init__(self, data)
-        self.retrieve_status = data
 
-    def render_started(self, ctx, data):
-        started_s = render_time(data.get_started())
-        return started_s
+# Renders "/status/retrieve-%d".
```

Comment:
docstring instead of comment would be better

Reply:
Made the change in 62760c0.